src/commands/upload_file.ts

```diff
@@ -121,6 +123,32 @@ function getSingleFile(parameters: ParametersHelper, parentFolderId: FolderID):
 	return [singleParameter];
 }
 
+async function getRemoteFile(
+	parameters: ParametersHelper,
+	parentFolderId: FolderID
+): Promise<UploadPathParameter[] | undefined> {
+	const remoteFilePath = parameters.getParameterValue(RemotePathParameter);
+
+	if (!remoteFilePath) {
+		return undefined;
+	}
+
+	const tempFolder = getTempFolder();
+
+	const localFilePath = await download(remoteFilePath, tempFolder);
```

Comment:
It occurs to me that we need some sensible requirements around choosing a file name when none is provided. Let's take to Slack as a team to hash something out. And really curious what @agsuy and @GiMartinezz have in mind.